.github/pull_request_template.md

```diff
@@ -22,12 +22,12 @@ _List any referenced GitHub issues. If PR references tickets in other systems (e
 
 _Make sure to use keywords to link this PR to GitHub issues ([GitHub Docs](https://docs.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword))._
 
-_Example:_ Closes https://github.com/example/repo/issues/123.
+_Example:_ Closes repo/issues#123.
```

Comment:
Will this shortened form work for repos across the `fossas` org?

Reply:
Yep.  Edit this comment and see how fossas/team-analysis#1 is rendered.  I've been doing it on PR's for a few months now.  It actually works across GH, not just within an org.

Reply:
I think it should be `org/repo#number` in the example text, though